spyder/utils/environ.py

```diff
@@ -38,6 +40,34 @@ def listdict2envdict(listdict):
     return listdict
 
 
+def clean_env(env_vars):

```

Comment:
I created this helper, we could probably use it also on the ipython plugin @ccordoba12 ?